packages/web-components/src/components/rux-tabs/rux-tab/rux-tab.scss

```diff
@@ -64,4 +64,8 @@
     color: var(--color-text-interactive-default);
     opacity: var(--opacity-disabled);
     cursor: not-allowed;
+    &:hover {
```

Comment:
This is exactly what's in next right now, so I'm not sure why this says it changed?

Reply:
proof: https://github.com/RocketCommunicationsInc/astro/blob/next/packages/web-components/src/components/rux-tabs/rux-tab/rux-tab.scss#L63